cmd/virtual-kubelet/internal/commands/root/root.go

```diff
@@ -152,7 +153,7 @@ func runRootCommand(ctx context.Context, s *provider.Store, c Opts) error {
 		leaseClient = client.CoordinationV1beta1().Leases(corev1.NamespaceNodeLease)
 	}
 
-	deployName := strings.Join([]string{"liqo", c.ClusterId}, "-")
+	deployName := strings.Join([]string{advop.VirtualKubeletPrefix, c.ClusterId}, "")
```

Comment:
I think that the advertisement-operator internals is the wrong place in which to put this const. Can we move it to a better place? (Maybe a shared pkg file)